### ISSUE TITLE
Harmonize `README`s across the ecosystem

### ISSUE THREAD

**Issue by IndrajeetPatil** [easystats/easystats#364]
Currently, `README` in every _easystats_ repo seems to be doing its own thing. Since all of them belong to the same ecosystem, we should try to follow a standard template for things which are common to all repos.

Here are some key features to standardize (and which currently differ across repos):

- Top-level badges 
- Installation instructions
- Badges in installation
- Citation 
- Documentation badges

Here are my preferences, but, of course, we can update these based on your feedback:

- Top-level badges: R-CMD-check, only total downloads, dependency count, codecov, lifecycle
- Installation instructions: Table with CRAN, r-universe, and pak options
- Badges in installation: None
- Citation: JOSS (if available), otherwise package citation
- Documentation badges: Only website badge

**Comment by bwiernik** [comment]
I prefer to list both the journal citation and package citation.

**Comment by mattansb** [comment]
I prefer avoiding tables for installation. Also, I see no need to promote `pak`. Users that use it will know what to do, and those that don't might be confused by the extra step.
Also, since we use r-universe, I think we can drop the `remotes` package as well.

**Comment by etiennebacher** [comment]
> I see no need to promote `pak`. Users that use it will know what to do, and those that don't might be confused by the extra step. 

Agreed and `pak` can have some issues on servers (I ran into this) so I think it's better to stick to `install.packages()`
